### PR TITLE
add truncatedLength as an option to card body

### PR DESCRIPTION
**Pull request by labradford**
Adds ability to pass in the truncated text length so the showmore toggle only shows according to what you want. Default is set to 300 still.

### CODE REVIEW

File: src/compounds/Item/CardBody.jsx [hunk 1]
```diff
@@ -26,13 +26,13 @@ const CardBody = ({ buttonLink, buttonProps, item, markdownDescriptionTruncated,
           </Card.Title>
             {orientation === 'horizontal' ? (
               <>
-                <div className='fw-light mh-300 overflow-auto mt-3'>
+                <div className='fw-light mh-300 overflow-auto mt-3 testing'>
```
Comment (alishaevn) [inline src/compounds/Item/CardBody.jsx:29]:
```suggestion
                <div className='fw-light mh-300 overflow-auto mt-3'>
```

File: src/compounds/Item/CardBody.jsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { Button, Card, Collapse } from 'react-bootstrap'
 import NextLink from '../../components/NextLink/NextLink'
 import LinkedButton from '../LinkedButton/LinkedButton'
 
-const CardBody = ({ buttonLink, buttonProps, item, markdownDescriptionTruncated, markdownDescriptionExtended, orientation, titleLink, withButtonLink, withTitleLink }) => {
+const CardBody = ({ buttonLink, buttonProps, item, markdownDescriptionTruncated, markdownDescriptionExtended, truncatedLength, orientation, titleLink, withButtonLink, withTitleLink }) => {
```
Comment (alishaevn) [inline src/compounds/Item/CardBody.jsx:7]: what do you think about changing this variable name to `truncateAt`? feels a little more semantic.

also, can you move this to after "titleLink"? the variables are alphabetized.

**Review by alishaevn — CHANGES_REQUESTED** [review]

File: src/compounds/Item/CardBody.jsx (hunk 2 above)
Comment (labradford) [reply src/compounds/Item/CardBody.jsx:7]: Summer called it cutOffIndex. Do you think it makes sense to use that?

Comment (alishaevn) [reply src/compounds/Item/CardBody.jsx:7]: I feel like `truncateAt` is even more semantic than `cutOffIndex`...but I'm fine with either.

**Review by alishaevn — APPROVED** [review]

**Pull request merged**